package.nix

```diff
@@ -1,196 +1,207 @@
 # dependencies
-{ nixos-generators
-, nixpkgs
-
-# pkgs
-, lib
-, mount
-, umount
-
-# configuration
-, linuxSystem
-, onDemand # enable launchd socket activation
+{
+  nixos-generators,
+  nixpkgs,
+  # pkgs
+  lib,
+  mount,
+  umount,
+  # configuration
+  linuxSystem,
+  debug ? false,
```

Comment:
Similar to `extraConfig` below, I have much less compunction to having this option at the image/package level.  I think I would like to rename it something a little more obviously "dangerous", though.

Reply:
Resolved in: https://github.com/cpick/nix-rosetta-builder/commit/a44b35977d8d653c4fad2019067367db260993bc